### DISCUSSION TITLE
UQL Jsonata - filter variables that have list of ints

### DISCUSSION THREAD

**Discussion by chicago-joe** [grafana/grafana-infinity-datasource#572] [Q&A]
Hi guys, absolutely love this library and the UQL examples provided by @yesoreyeram. 
Currently, I have a dashboard with a dozen variables that are parent-child-child-x...- format.

I'm able to get the variable "frontYear" as a list using the following UQL:
```
parse-json
| jsonata "*[exchange in [${exchange:singlequote}]][]"
| jsonata "*[exchangeBaseSymbol in [${exchangeBaseSymbol:singlequote}]][]"
| project "frontYear"
```
This is returned with the options "Multi-select" and "All" enabled.

Now, I want to create another dashboard variable that depends on the above + a filter on "frontYear", like so:
```
parse-json
| jsonata "*[exchange in [${exchange:singlequote}]][]"
| jsonata "*[exchangeBaseSymbol in [${exchangeBaseSymbol:singlequote}]][]"
| jsonata "*[frontYear in [${frontYear:singlequote}]][]"
| project "expYear"
```

However, the only value returned is "All". Does anyone know how I can filter on frontYear as well in this case?

**Comment by yesoreyeram** [comment]
Can you provide sample data and details about what you want to achieve? Without data, it is very hard to explore the solution
